packages/gitgraph-react/src/Gitgraph.tsx

```diff
@@ -101,11 +106,10 @@ class Gitgraph extends React.Component<GitgraphProps, GitgraphState> {
   public render() {
     return (
       <svg ref={this.$graph}>
-        {/* Translate graph left => left-most branch label is not cropped */}
+        {/* Translate graph left => left-most branch label is not cropped (horizontal) */}
         {/* Translate graph down => top-most commit tooltip is not cropped */}
         <g transform={`translate(${BranchLabel.paddingX}, ${Tooltip.padding})`}>
           {this.renderBranchesPaths()}
-          {this.renderBranchesLabels()}
```

Comment:
These were moved along other commit elements (message & tags) so it's easier to position everyone from the same origin.